CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Notable changes to this project are documented in this file. The format is based
 Breaking changes:
 - Update project and deps to PureScript v0.15.0 (#90 by @JordanMartinez)
 - Drop deprecated `MonadZero` instance (#90 by @JordanMartinez)
+- Change associativity of `withError` operator to match `Control.Alt` (#92 by @thomashoneyman)
```

Comment:
This needs to be updated to say the `prec` was dropped from 3 to 4 to prevent compiler error due to `<|>` change (or something like that).